packages/dev/package.json

```diff
@@ -13,14 +13,12 @@
   "scripts": {
     "tsc": "tsc",
     "watch-tsc": "tsc -w",
-    "truffle-compile": "truffle compile",
     "lint": "eslint -f unix .",
     "lint-fix": "yarn lint --fix",
-    "test": "lerna run tsc && truffle --compile-all --network npmtest test",
+    "solpp": "yarn --cwd=\"../..\" solpp",
+    "test": "yarn solpp && truffle --compile-all --network npmtest test",
```

Comment:
just to make sure: how do you test & compile incrementally ?

Reply:
Well, the created/modified timestamps on files are carried over, so truffle takes care of compiling incrementally. It is just important to run a correct yarn script so that `solpp` is invoked.

Reply:
So you must you a wrapper script and not use "truffle test" directly

Reply:
> So you must you a wrapper script and not use "truffle test" directly

Absolutely correct.